File: java/dayOne/levelTwo/TaskThree.java
import java.util.Arrays;

public class TaskThree {

    public static int [] swappingOfZeros(int [] numbers){
        
            
           int [] swappedNumbers = new int [numbers.length];
   
                int index = 0;
            for(int count = 0; count < numbers.length; count++){
            
                    if(numbers[count] != 0){
                       swappedNumbers[index] = numbers[count]; 
                        index++;
                        }

                    }
                 
                 return swappedNumbers;
                                 
    }

}
 

File: python/day_one/level_two/task_three.py


def swapping_of_zeros(numbers):

    new_list = []
    
    count = 0
    for number in numbers:
        if number != 0:
            new_list.append(number)
                
        elif number == 0:
            count +=1

    for zero in range(count):
        new_list.append(0)                
              
    return new_list

